### PR TITLE
#1109 Improve Pit Coverage

### PR DESCRIPTION
**Pull request by goldbal330**
Signed-off-by: Alexander Goldberg <alexander.goldberg@bnymellon.com>

### CODE REVIEW

File: unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/MaxProcedureTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021 Goldman Sachs and others.
```
Comment (prathasirisha) [inline unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/MaxProcedureTest.java:2]: This needs to be updated.

**Review by prathasirisha — CHANGES_REQUESTED** [review]
Added one minor comment on copyright update.

**Comment by prathasirisha** [comment]
@goldbal330 Thank you for your contribution. 😃 

One last comment, could you please update the commit message to reflect new tests are being added. This will help us a lot when we are preparing releasing notes. Commit message guidelines available [here](https://github.com/eclipse/eclipse-collections/blob/master/CONTRIBUTING.md#commit-messages).

**Comment by nikhilnanivadekar** [comment]
My only comment on this would be that we have another PR covering same tests by a new contributor. In the spirit of promoting new contributions can we see if we can selectively merge those PRs first and then this one. Thoughts?

**Review by prathasirisha — APPROVED** [review]

**Pull request merged**